packages/ketchup/src/components/kup-echart/kup-echart.tsx

```diff
@@ -898,6 +942,20 @@ export class KupEchart {
     }
 
     #setTitle() {
+        function convertColorString(input: string): string {
```

Comment:
There is an utility for this operation in KupTheme.
If it's not complete, it should be enhanced but this shouldn't be done in a component.